R/wilcoxonsr_hodges_lehman.qmd

```diff
@@ -0,0 +1,118 @@
+---
+title: "Wilcoxon signed rank test"
+---
+
+### **Introduction**
+
+Wilcoxon signed-rank test is a non-parametric equivalent of paired Student's t-test. It is a rank test, designed for analyzing repeated measures or paired observations by a paired comparison (a type of location test) to assess whether their population means differ. *Ties* are when you have two observations with the same result. For example, in a 2-period cross-over study, you take the difference between result on Treatment A minus result on Treatment B and find that two or more subjects have the same difference.
```

Comment:
I think we could add after  " to assess whether their population means differ."  "Whilst it does not 'compare' means or medians for a set of paired data, it ranks the results on A and ranks the results on B, then compares if Prob(A>B) > Prob(B>A)."